image/cli/mascli/functions/update

```diff
@@ -112,6 +112,15 @@ function update() {
   echo_h4 "IBM Operator Catalog" "    "
   echo_reset_dim "Catalog Version ..................... ${COLOR_MAGENTA}${MAS_CATALOG_VERSION}"
 
+  # In SLS 3.8.0 registry was changed to icr.io/cpopen, however
+  # when updating from a previous version it would not automatically
+  # change the value from cp.icr.io/cp hence we need to patch it
+  if [[ "$MAS_CATALOG_VERSION" == "v8-230926-amd64" ]]; then
```

Comment:
is there harm in not having this check and just doing the update regardless as to if it is correct or not? There is the case of someone doing the update on the next catalog after this but not this one (so skipping this one).

Reply:
if this update happens on previous catalogs it will break SLS. I am going to release SLS 3.8.1 soon which will autofix this issue. So really we just need to for any catalog that has 3.8.0 in it.